### ISSUE TITLE
CodaClient library is not working after 25c76d96fc7d2735f2872312d7230cafcce3c45a release

### ISSUE THREAD

**Issue by xni** [MinaProtocol/mina#7392]
Hi!

After installing a new release I'm getting this issue:

```
>>> from CodaClient import Client
>>> c = Client()
>>> c.get_daemon_status()
Traceback (most recent call last):
  File "/root/mina-daemon/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 706, in urlopen
    chunked=chunked,
  File "/root/mina-daemon/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/root/mina-daemon/venv/lib/python3.6/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.6/http/client.py", line 1373, in getresponse
    response.begin()
  File "/usr/lib/python3.6/http/client.py", line 311, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.6/http/client.py", line 280, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response
```

I'm using this library to run my own commission strategies and SRE tools.

Thank you!

**Comment by garethtdavies** [comment]
This release broke the GraphQL API hence the errors. I think this is resolved by https://github.com/MinaProtocol/mina/commit/1e773552a966353161c6c559eaf6905785b19d20

**Comment by mrmr1993** [comment]
This is fixed by d9ff8a07a8dab375b7bf5b371f196862c3590910, which will be in the next release.

**Comment by shimkiv** [comment]
Closing this issue as it is `stale` for a long time.
